tutorials/classical_simulation/Simulating with SPAM errors.ipynb

```diff
@@ -0,0 +1,335 @@
+{
```

Comment:
<p>We act on this atom with a Constant Pulse, such that its state oscillates between the ground and excited Rydberg states (Rabi oscillations):</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>